### ISSUE TITLE
Failed pip install on Windows 10

### ISSUE THREAD

**Issue by TheKursta** [jeremymcrae/liftover#6]
Using provided pip command the command window throws following error:
```bash
Building wheels for collected packages: liftover
  Building wheel for liftover (PEP 517) ... error
    ERROR: Command errored out with exit status 1:                                                                                                                
    command: 'C:\Users\TheKursta\anaconda3\envs\bioinfo\python.exe' 'C:\Users\TheKursta\anaconda3\envs\bioinfo\lib\site- 
    packages\pip\_vendor\pep517\_in_process.py' build_wheel 'C:\Users\THEKUR~1\AppData\Local\Temp\tmpx4i6tpd4'
   cwd: C:\Users\TheKursta\AppData\Local\Temp\pip-install-bb5j7_11\liftover_39538f844b6d4b05ad7649ef4a66117e
   Complete output (12 lines):                                                                                                                                  
   running bdist_wheel                                                                                                                                          
   running build                                                                                                                                                
   running build_py                                                                                                                                             
   creating build                                                                                                                                               
   creating build\lib.win-amd64-3.8                                                                                                                            
   creating build\lib.win-amd64-3.8\liftover                                                                                                                    
   copying liftover\download_file.py -> build\lib.win-amd64-3.8\liftover                                                                                        
   copying liftover\lifter.py -> build\lib.win-amd64-3.8\liftover                                                                                               
   copying liftover\__init__.py -> build\lib.win-amd64-3.8\liftover                                                                                             
   running build_ext                                                                                                                                            
   building 'liftover.chain_file' extension                                                                                                                     
   error: Don't know how to compile src/gzstream/gzstream.C                                                                                                     
    ----------------------------------------                                                                                                                     
    ERROR: Failed building wheel for liftover                                                                                                                 
    Failed to build liftover                                                                                                                                     
    ERROR: Could not build wheels for liftover which use PEP 517 and cannot be installed directly   
```

**Comment by jeremymcrae** [comment]
Thanks for the report! I couldn't figure out an easy fix for this unfortunately. The immediate issue of `Don't know how to compile src/gzstream/gzstream.C` can be fixed by renaming to `gzstream.cpp` (and renaming same path in `setup.py`). But then it complains about missing `zlib.h`. It should be possible to work around that by pointing to zlib.h in your conda libraries, but I couldn't figure out the commands to make this work on my own conda environment.

**Comment by MattCloward** [comment]
I am also getting the same error mentioned above in the latest version of liftover (v1.1.11). I am using python 3.9.7.
Are there any fixes for this issue?

**Comment by jeremymcrae** [comment]
I think I've fixed this now, as of ada1bc639711addf555d34a9e41e9669d162e4fe. At least, it installs and runs without errors under windows on a local machine, and via github actions. Can you try installing version 1.1.12 again, and see if that works?

**Comment by jeremymcrae** [comment]
Closing, as I think this is fixed, but reopen if there still are problems.
